projects/packages/search/src/dashboard/components/record-meter/test/record-count.test.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * @jest-environment jsdom
+ */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+/**
+ * Internal dependencies
+ */
+import { RecordCount } from 'components/record-meter/record-count';
+
+describe( 'record count', () => {
+	test( 'outputs correct text', () => {
+		render( <RecordCount recordCount={ 20 } planRecordLimit={ 100 } /> );
+
+		expect(
+			screen.getByText( '20 records indexed out of the 100 allotted for your current plan' )
```

Comment:
This could be a fragile test because it relies on that exact text to pass. If we tweaked the wording in the future it would fail. For example, removing "the" from the phrase.

A couple of ideas to make it more resilient:
* Look for a partial string like "x allotted" or "x records" which is less likely to change
* Wrap the numbers each in a span and get the values using the class name instead? e.g `<span className="record-count__current-count">`